### ISSUE TITLE
Object detection Accuracy. 

### ISSUE THREAD

**Issue by Annieliaquat** [tensorflow/models#10660]
Hi, please can someone help me to find accuracy in Tensorflow object detection API. I want to compare my model SSD mobnet-FPNlite with other models but other papers have accuracy as a matrics. Tensorflow object detection API outputs mAP and recall only as a matrics. Please I really want the accuracy to be displayed

**Comment by snowMan108** [comment]
To evaluate object detection models like R-CNN and YOLO, the mean average precision (mAP) is used. The mAP compares the ground-truth bounding box to the detected box and returns a score. The higher the score, the more accurate the model is in its detections.

**Comment by Annieliaquat** [comment]
> To evaluate object detection models like R-CNN and YOLO, the mean average precision (mAP) is used. The mAP compares the ground-truth bounding box to the detected box and returns a score. The higher the score, the more accurate the model is in its detections.

Thank you so much.

**Comment by raulperezalejo** [comment]
@snowMan108 how do you identify Overfitting? I used to do this with the training history and the accuracy between the training and the validation. Not sure if there is another way, so far have not been able to graph the training history
